core/cbor_blake/ipld_version.hpp

```diff
@@ -8,6 +8,7 @@
 #include "storage/ipfs/datastore.hpp"
 
 namespace fc {
+  // TODO(turuslan): refactor Ipld to CbIpld
   struct IpldProxy : Ipld {
```

Comment:
Why not IpldVersion if you changed the file name?

Reply:
I thought `Proxy` is more concise (what else can it do with just `Ipld` pointer)